src/contracts/types/LockTypes.sol

```diff
@@ -1,37 +1,34 @@
 //SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.22;
 
-struct EscrowKey {
+struct Lock {
+    address activeEnvironment;
+    ExecutionPhase phase;
+    uint32 callConfig;
+}
+
+struct Context {
     address executionEnvironment; // not packed
     bytes32 userOpHash; // not packed
-    address bundler; // not packed
-    address addressPointer;
+    address bundler;
```

Comment:
hmm. good point. I was intending to stick with the naming scheme from the white paper, but technically this is the "caller" and the actual bundler can be discovered with a `tx.origin` call.

Reply:
We've been using "bundler" as the term for metacall caller quite ubiquitously in the code so far, so feels normal to me. But maybe we should change this now if you think it will be confusing for dapps.